src/transformers/training_args.py

```diff
@@ -572,6 +572,14 @@ class TrainingArguments:
         default="",
         metadata={"help": "Used by the SageMaker launcher to send mp-specific args. Ignored in Trainer"},
     )
+    sparse_log_frequency: float = field(
```

Comment:
So by default, this would mean e.g.
with batch_size 100:
we would log data at every 100*0.1 (10th) step of the epoch, right?
nit: I do not like the naming of this variable. How about something more informative like `modifier_log_frequency`?

Reply:
I don't think it relates to the `batch_size`, my understanding is that if this field is 0.1 then we would log data whenever 10% of an epoch completes, like we would log at 10%, 20%, 30% ... 100% of the epoch.

Reply:
@dbogunowicz good point, renamed to your suggestion. Also, what @rahul-tuli said is correct, the log frequency is independent of batch size. As soon as `log_frequency` number of epochs passes, it logs at the next step